crates/libcontainer/src/process/container_init_process.rs

```diff
@@ -757,6 +763,40 @@ fn set_supplementary_gids(
     Ok(())
 }
 
+/// set_io_priority set io priority
+fn set_io_priority(syscall: &dyn Syscall, io_priority_op: &Option<LinuxIOPriority>) -> Result<()> {
+    match io_priority_op {
+        Some(io_priority) => {
+            let io_prio_class_mapping: HashMap<_, _> = [
+                (IOPriorityClass::IoprioClassRt, 1i64),
+                (IOPriorityClass::IoprioClassBe, 2i64),
+                (IOPriorityClass::IoprioClassIdle, 3i64),
+            ]
+            .iter()
+            .map(|(class, num)| (serde_json::to_string(&class).unwrap(), *num))
```

Comment:
Nit: don't unwrap please. You can use map_err to return an error if you have to.